library/src/main/java/com/thesurix/gesturerecycler/GestureAdapter.java

```diff
@@ -73,21 +73,29 @@
          * Called when view holder item has pending drag gesture.
          * @param viewHolder dragged view holder item
          */
-        void onStartDrag(GestureViewHolder viewHolder);
+        void onStartDrag(@NonNull GestureViewHolder viewHolder);
     }
 
-    private static final int INVALID_DRAG_POS = -1;
+    /**
+     * The maximum size of undo stack to allocate.
+     * Some VMs reserve some header words in an array.
+     * Attempts to allocate larger arrays may result in
+     * OutOfMemoryError: Requested array size exceeds VM limit
+     * */
+    public final static int MAX_UNDO_STACK_SIZE = Integer.MAX_VALUE - 8;
```

Comment:
I don't understand why we need this constant, it is only used in JD. There is no validation etc. Can you give me some information about it?